### ISSUE TITLE
Enforce `.get` with dicts over `in` checks

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#998]
# Rule request

## Thesis

When we see a code like:

```python
if 'inr' in currency_map:
    indian_currency_name = currency_map['inr']
else:
    indian_currency_name = 'undefined'
```

We should raise a refactoring violation. That's how it should be:

```python
currency_map = {'usd': 'US Dollar'}
indian_currency_name = currency_map.get('inr', 'undefined')
```

The second version is more pythonic.

Origin: https://deepsource.io/blog/8-new-python-antipatterns

**Comment by sobolevn** [comment]
https://github.com/PyCQA/pylint/blob/master/pylint/checkers/refactoring.py#L224
